packages/ui-app/src/AddressInfo.tsx

```diff
@@ -130,6 +139,46 @@ class AddressInfo extends React.PureComponent<Props> {
       </div>
     );
   }
+
+  private renderRedeemButton () {
+    const { staking_info, t } = this.props;
+
+    return (staking_info && staking_info.controllerId && <TxButton
+      accountId={staking_info.controllerId.toString()}
+      className='iconButton'
+      icon='lock'
+      size='small'
+      isPrimary
+      key='unlock'
+      params={[]}
+      tooltip={t('Redeem these funds')}
+      tx='staking.withdrawUnbonded'
+    />
+    );
+  }
+
+  private renderUnlocking () {
+    const { staking_info, t } = this.props;
+
+    return (
+      staking_info &&
+      staking_info.unlocking &&
+      staking_info.unlocking.map(({ remainingBlocks, value }, index) => (
+        <div key={index}>
+          {formatBalance(value)}
+          <Icon
+            name='info circle'
+            data-tip
+            data-for={`controlled-trigger${index}`}
```

Comment:
Maybe just `unlocking-trigger-${index}` (I believe we use `controlled-trigger-` in places, so this would just be slightly safer?